apps/src/codebridge/ControlButtons/index.tsx

```diff
@@ -53,7 +55,12 @@ const ControlButtons: React.FunctionComponent = () => {
   const hasSubmitted = useAppSelector(
     state => getCurrentLevel(state)?.status === LevelStatus.submitted
   );
-  const disableRunAndTest = loading || (isPredictLevel && !hasPredictResponse);
+  const isStartMode = getAppOptionsEditBlocks() === START_SOURCES;
```

Comment:
Minor: not required for this change, but wondering if it would be good to have a function in `utils` that just returns `isStartMode` directly

Reply:
good call, it probably would. I think we set it up like this since we would probably need to get different edit blocks values, but we check start sources enough that it would be good to have a dedicated method.